### PR TITLE
Fyle 1uvx789 : Fixed change top Information Box to Alert Box in expense/mileage/per diem pages

### PR DESCRIPTION
**Pull request by siddharth0-0**
Screenshot_20211119-150631_Fyle Staging

image

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.html [hunk 1]
```diff
@@ -80,8 +80,11 @@
 
           <ng-container *ngIf="isAmountCapped$|async">
             <div class="add-edit-expense--critical-policy" *ngIf="!(isCriticalPolicyViolated$|async)">
-              Claimed amount {{etxn.tx.user_amount | number: '1.0-2'}} was capped to {{etxn.tx.amount | number:
-              '1.0-2'}} due to policy <span *ngIf="isAmountDisabled$|async"> and cannot be edited.</span>
+              <mat-icon class="add-edit-expense--critical-policy-icon" svgIcon="fy-info"></mat-icon>
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.html:83]: this is not a critical-policy-icon, can reuse the same class of split expense

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 2]
```diff
@@ -405,10 +405,23 @@
   }
 
   &--critical-policy {
-    border-radius: 6px;
-    background: #f2f2f2;
-    padding: 8px 16px;
     margin: 8px 0;
+    display: flex;
+    border: 1px solid #dfdfe2;
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:410]: use color variable

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 3]
```diff
@@ -405,10 +405,23 @@
...
+    border-radius: 8px;
+    padding: 12px 8px;
+  }
+
+  &--critical-policy-icon {
```
Comment (Dimple16) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:415]: use the same class used for split expense info

File: src/app/fyle/add-edit-mileage/add-edit-mileage.page.scss [hunk 4]
```diff
@@ -249,11 +249,23 @@
...
-    margin-bottom: 16px;
-    margin-top: 8px;
+    margin: 8px 0;
+    display: flex;
+    border: 1px solid #dfdfe2;
+    border-radius: 8px;
+    padding: 12px 8px;
+  }
+
+  &--critical-policy-icon {
```
Comment (Dimple16) [inline src/app/fyle/add-edit-mileage/add-edit-mileage.page.scss:259]: same as above

File: src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.scss [hunk 5]
```diff
@@ -289,10 +289,23 @@
...
-    padding: 8px 16px;
-    margin-bottom: 16px;
+    margin: 8px 0;
+    display: flex;
+    border: 1px solid #dfdfe2;
+    border-radius: 8px;
+    padding: 12px 8px;
+  }
+
+  &--critical-policy-icon {
```
Comment (Dimple16) [inline src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.scss:299]: same as above

**Review by Dimple16 — CHANGES_REQUESTED** [review]
few comments

**Review by Dimple16 — APPROVED** [review]

**Review by Julias0 — APPROVED** [review]

**Pull request merged**